index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>favour's porfolio page</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header id="toolbar">
+      <img
+        id="logoimage"
+        src="images/image_geometry_1.png"
+        alt="My portfolio's logo"
+      />
+      <img id="menuimage" src="./images/ic_menu.svg" alt="menu icon" />
+    </header>
+    <section class="headline">
+      <h1 class="name">Favour <br />Ezeugwa</h1>
+      <p class="bio">
+        Hello! I am a software developer! I can help you build a product,
+        feature or website. Taka a look of my works. If you like what you see
+        and have a project you need coded, don't hesitate and contact me.
+      </p>
+      <div class="profilelin">
+        <a class="toprofile" href="https://github.com/Favourezeugwa"
+          >Start collaboration</a
+        >
+      </div>
+    </section>
+    <div>
+      <img
+        id="Geometry2"
+        src="./images/image_geometry_2.png"
+        alt="porfolio theme"
+      />
+    </div>
```

Comment:
- Kindly try to stick as much as possible to the Fima design, which includes:
   - [ ] Adjusting the size of the background image to match the Figma design.
   - [ ] Implementing the "Start collaboration" to be a `button` tag instead of a `div`, and kindly make sure to implement all the button interactions (hover, pressed, ...etc).

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/164487531-bafa8d3d-a8e5-474f-8a49-a45fe23adede.png) | ![image](https://user-images.githubusercontent.com/62290545/164487769-185df1dd-1e79-4690-87f9-3506512f2499.png) |